### PR TITLE
Implement structure for Onboarding data

### PR DESCRIPTION
**Pull request by Doc94**
https://github.com/discord/discord-api-docs/commit/e986a12bd3eceeec3fea10bc957a59fb9e53ef29

### CODE REVIEW

File: src/main/java/discord4j/discordjson/json/OnboardingData.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        return ImmutableOnboardingData.builder();
+    }
+
+    @JsonProperty("guild_id")
+    Id guildId();
+
+    List<OnboardingPromptData> prompts();
+
+    @JsonProperty("default_channel_ids")
+    List<Id> defaultChannelsIds();
```
Comment (quanticc) [inline src/main/java/discord4j/discordjson/json/OnboardingData.java:25]: Let's use `defaultChannelIds` instead

**Review by quanticc — CHANGES_REQUESTED** [review]
Thanks!

Can you include a test with the example json from the docs?

https://github.com/discord/discord-api-docs/commit/e986a12bd3eceeec3fea10bc957a59fb9e53ef29#diff-6a43fc10e10de61f31b90ab22421ef0612d0ea75e3cd53891c6393a6fb84872dR570-R623

File: src/main/java/discord4j/discordjson/json/OnboardingData.java (hunk 1 above)
Comment (Doc94) [reply src/main/java/discord4j/discordjson/json/OnboardingData.java:25]: okay i based this in another fields with the same thing about the channel in singular and ids. but changing.

**Comment by Doc94** [comment]
> Thanks!
> 
> Can you include a test with the example json from the docs?
> 
> [discord/discord-api-docs@e986a12#diff-6a43fc10e10de61f31b90ab22421ef0612d0ea75e3cd53891c6393a6fb84872dR570-R623](https://github.com/discord/discord-api-docs/commit/e986a12bd3eceeec3fea10bc957a59fb9e53ef29#diff-6a43fc10e10de61f31b90ab22421ef0612d0ea75e3cd53891c6393a6fb84872dR570-R623)

added also i change a little the example because the format was wrong (related to https://github.com/discord/discord-api-docs/pull/6063)

**Review by quanticc — APPROVED** [review]

**Pull request merged**